sdcm/utils/cloud_monitor/resources/static_ips.py

```diff
@@ -84,6 +98,15 @@ def get_gce_static_ips(self):
                 eip.owner = cloud_instances_by_name[eip.used_by].owner
         self.all.extend(self["gce"])
 
+    def get_azure_static_ips(self):
+        # all azure ip's are external resources and we pay for it
+        query_bits = ["Resources", "where type =~ 'Microsoft.Network/publicIPAddresses'",
```

Comment:
we have code that create static ips for Azure ? in which test case it's being used ?

Reply:
"all azure ip's are external" - we don't have internal only addresses in Azure ?

Reply:
before creating VM, external IP must be provided (we could skip it, but then VM wouldn't be accessible from outside).
This external IP lives own life - deleting vm may not delete IP address.

Comment in that terms is not correct, should be:
` all azure public ip's are external resources and we pay for it`
I'll fix it